tools/systematic.py

```diff
@@ -382,6 +385,7 @@ def scaleTopMassSystematic( upper_uncertainty, lower_uncertainty, topMasses, top
 
 def get_sign(central, upper, lower, upper_variation, lower_variation):
     '''
+    @deprecated
```

Comment:
That is not how you apply a decorator ;).
The decorator goes directly above `dev get_sign`:
http://www.learnpython.org/en/Decorators